tests/integration/src/test/java/org/apache/pulsar/tests/integration/io/sources/debezium/DebeziumMongoDbSourceTester.java

```diff
@@ -50,7 +50,7 @@ public DebeziumMongoDbSourceTester(PulsarCluster cluster) {
         sourceConfig.put("mongodb.password", "dbz");
         sourceConfig.put("mongodb.task.id","1");
         sourceConfig.put("database.whitelist", "inventory");
-        sourceConfig.put("database.history.pulsar.service.url", pulsarServiceUrl);
+        sourceConfig.put("history.database.pulsar.service.url", pulsarServiceUrl);
```

Comment:
I thought its should start with `DatabaseHistory.CONFIGURATION_FIELD_PREFIX_STRING` which is `CONFIGURATION_FIELD_PREFIX_STRING = "database.history."`
debezium does 
```
Configuration dbHistoryConfig = config.subset(DatabaseHistory.CONFIGURATION_FIELD_PREFIX_STRING, false)
```

Reply:
Yeah, I just updated them.